config/garbage-collector/manager.yaml

```
@@ -0,0 +1,61 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: garbage-collection-manager
+  labels:
+    control-plane: garbage-collection-manager
+    app.kubernetes.io/name: cronjob
+    app.kubernetes.io/instance: garbage-collection-manager
+    app.kubernetes.io/component: garbage-collector
+    app.kubernetes.io/created-by: k8s-backup-operator
+    app.kubernetes.io/part-of: k8s-backup-operator
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  schedule: "{{ .Values.retention.garbageCollectionCron }}"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: "garbage-collection-manager"
+          restartPolicy: OnFailure
+          securityContext:
+            runAsNonRoot: true
+            seccompProfile:
+              type: RuntimeDefault
+          volumes:
+            - name: retention-config
+              configMap:
+                name: k8s-backup-operator-retention
+                optional: true
+          containers:
+          - args:
+            - gc
+            - --strategy="{{ .Values.retention.strategy }}"
```

```suggestion
            - --strategy={{ .Values.retention.strategy }}
```
The quotes are part of the flag and should be removed here or while parsing the flag